src/server/schema/resolvers/queries/getLife.ts

```diff
@@ -0,0 +1,17 @@
+import { ObjectId } from 'mongodb';
+import { getDatabaseContext } from '../../../database';
+import { GraphQLQueryResolvers } from '../definitions';
+
+const query: GraphQLQueryResolvers['getLife'] = async (root, { id }) => {
+    const { collections } = await getDatabaseContext();
+    const lifeID = new ObjectId(id);
```

Comment:
`id` here is of type `ObjectId` already. No need to do this

Reply:
I see! Is this because of the type resolver created for `ObjectID` in `/src/server/schema/resolvers/types/ObjectID.ts`

Reply:
Yup. It is a Custom GraphQL Type.
So when this data is received, we are already converting it to be an `ObjectId`